src/components/CalendarEvent.js

```diff
@@ -84,8 +84,20 @@ class CalendarEvent extends Component {
             <div className="calendar-event-activity-wrapper">
               {rows}
             </div>
-            <Button>Muokkaa</Button>
-            <Button>Poista</Button>
+            <div className="calendar-event-button-wrapper">
+              <EditEvent
+                buttonClass="calendar-button"
+                data={event}
+                source={this.handleClose}
```

Comment:
Are these lines unnecessary?

Reply:
The EditEvent-component requires those props or else it logs errors.
Buttonclass is unnecessary but requires refactoring in EditEvent.js by defining the class there.
data is a reference to the event we want to edit, absolutely necessary
handleClose is also redundant for the popover, but required in the list view edit popup, since it "unexpands" the event. However without the prop we get console warnings

Reply:
I don't think replacing a separately defined empty function with an inline empty function is a worthwhile change.

Instead I suggest changing EditEvent to gracefully handle a missing handleClose prop. I can do this tomorrow unless another solution is better.

Reply:
Sounds good